src/main/java/com/example/springcrud/service/ProductService.java

```diff
@@ -47,9 +55,9 @@ public void updateProduct(Integer productId, String title, String description, D
         if (description != null && description.length() > 0 && !Objects.equals(product.getDescription(), description)) {
```

Comment:
In loc de **description.length() > 0** ai putea folosi **!description.isEmtpy()**, ce face acelasi lucru. La fel este metoda **isBlank()**, ce da true daca stringul e doar cu log gol, de genul "   ".